src/views/accounts/RiskDetect/AccountRiskList.vue

```diff
@@ -162,7 +162,7 @@ export default {
 
       if (!payload) return
 
-      const queryParams = `?risk=${payload}`
+      const queryParams = `&risk=${payload}`
 
       if (queryParams) {
         this.tableConfig.url += queryParams
```

Comment:
It looks like there are no changes between the exported function definitions for the two code snippets in question. The code is self-contained in both cases, and they're doing essentially what they're supposed to without including additional logic that wouldn't change depending on the environment (like HTTP authentication). However, I'm not aware of any common best practices regarding this exact piece of code. You might want to ask it out in more specific contexts about how exactly you'd use something like this.